mentors/ruby/c.json

```diff
@@ -47,4 +47,12 @@
     "avatar_url": "https://avatars0.githubusercontent.com/u/987305",
     "bio": "I've worked with Ruby professionally for over 10 years now. The Ruby community is so incredible and I'm happy to be giving back as a mentor."
   }
```

Comment:
Please add a `,` after `}`

Reply:
Hopefully, the typo has been resolved. 😄